app/formats.js

```diff
@@ -0,0 +1,30 @@
+export default {
```

Comment:
do we have anything in the addon that has a hard dep on the ember-intl addon? if not i'd probably just rip it out, along with all these related config files.